Core/Systems/ModPlayers/UniversalBuffingPlayer.cs

```diff
@@ -10,6 +10,7 @@ internal class UniversalBuffingPlayer : ModPlayer
 
 	public override void PostUpdateEquips()
 	{
+		Player.lifeRegen = Player.lifeRegen;
```

Comment:
What's the point of this?

Reply:
Whoops! Some debug I forgot to get rid of. I'll remove it lol